macosx/InfoWindowController.mm

```diff
@@ -91,12 +91,31 @@ - (void)awakeFromNib
     window.becomesKeyOnlyIfNeeded = YES;
 
     //set tab tooltips
-    [fTabMatrix setToolTip:NSLocalizedString(@"General Info", "Inspector -> tab") forCell:[fTabMatrix cellWithTag:TAB_GENERAL_TAG]];
-    [fTabMatrix setToolTip:NSLocalizedString(@"Activity", "Inspector -> tab") forCell:[fTabMatrix cellWithTag:TAB_ACTIVITY_TAG]];
-    [fTabMatrix setToolTip:NSLocalizedString(@"Trackers", "Inspector -> tab") forCell:[fTabMatrix cellWithTag:TAB_TRACKERS_TAG]];
-    [fTabMatrix setToolTip:NSLocalizedString(@"Peers", "Inspector -> tab") forCell:[fTabMatrix cellWithTag:TAB_PEERS_TAG]];
-    [fTabMatrix setToolTip:NSLocalizedString(@"Files", "Inspector -> tab") forCell:[fTabMatrix cellWithTag:TAB_FILE_TAG]];
-    [fTabMatrix setToolTip:NSLocalizedString(@"Options", "Inspector -> tab") forCell:[fTabMatrix cellWithTag:TAB_OPTIONS_TAG]];
+    [fTabs.cell setToolTip:NSLocalizedString(@"General Info", "Inspector -> tab") forSegment:TAB_GENERAL_TAG];
+    [fTabs.cell setToolTip:NSLocalizedString(@"Activity", "Inspector -> tab") forSegment:TAB_ACTIVITY_TAG];
+    [fTabs.cell setToolTip:NSLocalizedString(@"Trackers", "Inspector -> tab") forSegment:TAB_TRACKERS_TAG];
+    [fTabs.cell setToolTip:NSLocalizedString(@"Peers", "Inspector -> tab") forSegment:TAB_PEERS_TAG];
+    [fTabs.cell setToolTip:NSLocalizedString(@"Files", "Inspector -> tab") forSegment:TAB_FILE_TAG];
+    [fTabs.cell setToolTip:NSLocalizedString(@"Options", "Inspector -> tab") forSegment:TAB_OPTIONS_TAG];
+
+    [fTabs setImage:[NSImage systemSymbol:@"info.circle"
+                             withFallback:@"InfoGeneral"]
+         forSegment:TAB_GENERAL_TAG];
+    [fTabs setImage:[NSImage systemSymbol:@"number.square"
+                             withFallback:@"InfoActivity"]
+         forSegment:TAB_ACTIVITY_TAG];
+    [fTabs setImage:[NSImage systemSymbol:@"antenna.radiowaves.left.and.right"
+                             withFallback:@"InfoTracker"]
+         forSegment:TAB_TRACKERS_TAG];
+    [fTabs setImage:[NSImage systemSymbol:@"person.fill"
```

Comment:
I'd not use the fill version here, since that doesn't match the others which are outlines

Reply:
Also, the same symbol as the one used in Peers preferences might make sense here, since they're the same name.